extensions/wikia/CommunityPage/templates/recentlyJoined.mustache

```diff
@@ -2,10 +2,10 @@
 	<h2>{{recentlyJoinedHeaderText}}</h2>
 	{{#members}}
 		<div class="avatar-container">
-			<a href="{{profilePage}}">{{{avatar}}}</a>
+			<a data-tracking="recently-joined-avatar" href="{{profilePage}}">{{{avatar}}}</a>
 		</div>
 	{{/members}}
 {{/haveMembers}}
 <span class="recently-joined">
-	<a href="#" id="viewAllMembers">{{allMembers}}</a>
+	<a data-tracking="show-modal" href="#" id="viewAllMembers">{{allMembers}}</a>
```

Comment:
In upcoming https://github.com/Wikia/app/pull/10399 we'll adding new entry point to open modal. What about changing the label to tell which entry point is clicked? e.g. show-modal-all